src/main/java/gift/product/domain/ProductDTO.java

```diff
@@ -0,0 +1,60 @@
+package gift.product.domain;
+
+public class ProductDTO {
+    Long id=0L;
```

Comment:
접근 제어자 + 라인 포매팅 + 가시성(protected) + 불필요한 setter 제거 반영되면 좋을 것 같아요ㅎㅎ
setter가 modelattribute 때문에 필요하다면 놔두세요!
참고로 Command + Option + L 누르면 라인 자동 정렬됩니다:)
```suggestion
    private Long id=0L;
```